docs/lambda.asciidoc

```diff
@@ -19,6 +17,20 @@ check out the <<api,API Reference>>.
 [float]
 [[lambda-installation]]
 ==== Installation
+Setting up your Lambda function is a two step process. First, you’ll
+need to install https://github.com/elastic/apm-aws-lambda/[the Elastic
```

Comment:
Thoughts on pointing that directly to the extension subdir: https://github.com/elastic/apm-aws-lambda/tree/main/apm-lambda-extension ?

Reply:
As-is, the link to the [repo](https://github.com/elastic/apm-aws-lambda/) doesn't provide much context. You also can't immediately see a docs directory or readme, so it doesn't seem very user-friendly. I think it would be a better user experience if we link to the documentation before linking to the repo itself. I also wonder if we should break installation of the AWS Lambda extension into its own step like Instrumenting your Lambda Handler is. For example, something like this:


```
[float]
[[lambda-installation]]
==== Installation

Setting up your Lambda function is a two-step process.

1. <<lambda-extension>>
2. <<lambda-instrumenting>>

First, https://github.com/elastic/apm-aws-lambda/blob/main/apm-lambda-extension/docs/aws-lambda-extension.asciidoc[install the Elastic
APM Lambda extension] into your Lambda environment. Elastic uses a Lambda
extension to forward data to APM Server in a way that does not interfere with the
execution of your Lambda function. 

[float]
[[lambda-extension]]
==== Install the Elastic APM Lambda extension

Elastic uses a Lambda extension to forward data to APM Server in a way that does not interfere with the
execution of your Lambda function.

See the https://github.com/elastic/apm-aws-lambda/blob/main/apm-lambda-extension/docs/aws-lambda-extension.asciidoc[installation documentation] to get started.

Once you have the extension installed in your environment, you can
configure your Lambda handler to instrument your function.

[float]
[[lambda-instrumenting]]
==== Instrumenting your Lambda Handler
```